api_tests/view_only_links/views/test_view_only_link_nodes.py

```diff
@@ -201,7 +201,7 @@ def test_set_node_not_component(self, app, user, project_two, url):
             auth=user.auth,
             expect_errors=True)
         assert res.status_code == 400
-        assert res.json['errors'][0]['detail'] == 'The node {0} cannot be affiliated with this View Only Link because the node you\'re trying to affiliate is not descended from the node that the View Only Link is attached to.'.format(
+        assert res.json['errors'][0]['detail'] == 'The node {} cannot be affiliated with this View Only Link because the node you\'re trying to affiliate is not descended from the node that the View Only Link is attached to.'.format(
             project_two._id)
```

Comment:
Unsure if this will lint
```suggestion
        assert (
            res.json['errors'][0]['detail'] == 
            f"The node {project_two._id} cannot be affiliated with this View Only Link because the node you're trying to affiliate is not descended from the node that the View Only Link is attached to."
        )
```